tripy/tripy/frontend/module/layernorm.py

```diff
@@ -43,7 +43,7 @@ class LayerNorm(Module):
     dtype: datatype.dtype
     r"""The data type used to perform the operation."""
 
-    normalized_shape: Tuple[int]
+    normalized_shape: Sequence[int]
```

Comment:
Note that `Tuple` specifies a fixed number of arguments. For sequences of arbitrary length, we should use `Sequence` instead.